cypress/integration/about.ts

```diff
@@ -0,0 +1,57 @@
+describe('The about page', () => {
+  it('has working link to comparator page', () => {
+    cy.visit('/')
+
+    cy.contains('Comparator').click()
+
+    cy.url().should('include', '/')
+  })
+})
+
+describe('The about page', () => {
+  it('has correct page title', () => {
+    cy.visit('/about')
+
+    cy.title().should(
+      'equal',
+      'About | Octoclairvoyant: Compare GitHub changelogs across multiple releases'
```

Comment:
I tried importing the `BRIEF_DESCRIPTION` constant from the `common.ts` file but couldn't get it working.
Maybe you can try with your hot-shot TypeScript and Cypress skills? 😉 

Link to file where the constant I want to import lives: https://github.com/Belco90/octoclairvoyant/blob/main/src/common.ts

Reply:
This seems an issue with our current TS setup, I'll try to take a look at our `tsconfig.json` during the week.

Reply:
As we want to use a hard-coded value instead of referring to the imported `const` I'll resolve this review comment.